internal/domain/power-cap.go

```diff
@@ -732,6 +734,23 @@ func doPowerCapTask(taskID uuid.UUID) {
 						break
 					}
 				}
+
+				// Convert PowerConsumedWatts to an int if not already (it's an interface{}
+				// type that can support ints and floats) - Needed for Foxconn Paradise,
+				// perhaps others in the future
+				for _, pwrCtl := range rfPower.PowerCtl {
+					if pwrCtl.PowerConsumedWatts != nil {
+						switch v := (*pwrCtl.PowerConsumedWatts).(type) {
+						case float64:	// Convert to int
+							*pwrCtl.PowerConsumedWatts = int(math.Round(v))
+							logger.Log.WithFields(logrus.Fields{"type": reflect.TypeOf(*pwrCtl.PowerConsumedWatts), "value": *pwrCtl.PowerConsumedWatts}).Errorf("Unexpected type/value detected for PowerConsumedWatts, setting to 0\n")
```

Comment:
Bad log statement. Copy and paste from the default case log.

Reply:
Good catch!  I had copy/pasted it there to make sure the default case log would log correctly if/when it ever gets hit.  I've pushed a change to resolve this.